icu4j/main/classes/core/src/com/ibm/icu/util/BasicTimeZone.java

```diff
@@ -600,6 +683,17 @@ public void getOffsetFromLocal(long date,
         throw new IllegalStateException("Not implemented");
     }
 
+    /**
+     * {@icu} Returns time zone offsets from local wall time.
+     * @draft ICU 69
+     * @provisional This API might change or be removed in a future release.
+     */
+    public void getOffsetFromLocal(long date,
```

Comment:
In Java, I prefer deleting/replacing the "int" version rather than delegating to it.

Reply:
^ I actually had a version which deleting int implementation completely, but reverted/squashed to match ICU4C implementation for now.

Reply:
@markusicu It looks I lost the changes by squashing commits and force push. I can work on it if it's really desired. I prefer to keep C and J synchronized as much as possible.

Reply:
Lost which changes? The changes that removed the Java int version? If that's too much trouble now, then please include a comment about Java in whatever ticket we have to remove the C++ int version in the next release.

Reply:
OK - dropping @internal / @deprecated getOffsetFromLocal taking int args for options from BasicTimeZone is easy - and can make it happen now. Chaning internal implementation not depending on these deprecated public constants - LOCAL_FORMER/LOCAL_STD, etc... requires much more changes (although it's mostly mechanical changes, just many places).
I can drop the method signature from ICU4J now.

Reply:
@markusicu - deleted the old getOffsetFromLocal from Java BasicTimeZone in the latest commit. Please check.